pyccel/ast/sysext.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+""" Module containing objects from the sys module understood by pyccel
+"""
+from .core import PyccelFunctionDef, Module
+from .internals import PyccelInternalFunction
+from .datatypes import NativeVoid
+from .internals import LiteralInteger
+
+class SysExit(PyccelInternalFunction):
+    """Represents a call to  sys.exit
+
+    Parameters
+    ----------
+
+    arg : PyccelAstNode (optional)
+        if arg.dtype is NativeInteger it will be used as the exit_code
+        else the arg will be printed to the stderror
+    """
+    __slots__ = ()
+    name      = 'exit'
+    _dtype     = NativeVoid()
+    _precision = -1
+    _rank      = 0
+    _shape     = None
+    _order     = None
+
+    def __init__(self, arg=None):
```

Comment:
I think this argument should be called `status`. Does this work if you translate:
```python
x = sys.exit(status=2)
```
?